src/viam/robot/client.py

```diff
@@ -53,6 +53,16 @@
 LOGGER = logging.getLogger(__name__)
 
 
+@dataclass
+class CloudMetadata:
+    """App-related information about the robot"""
+
+    primary_org_id: str
+    location_id: str
+    machine_id: str
+    machine_part_id: str
```

Comment:
I would rather use a `TypeAlias` here rather than creating a new type, especially since everything simply mirrors the `GetCloudMetadataResponse` anyway

Reply:
would making a `TypeAlias` expose the `robot_part_id` field on `GetCloudMetadataResponse` too? this is a duplicate of `machine_part_id` - we left it to prevent a breaking proto change

Reply:
Yes it would. So if you're trying to hide that field, then sure you can create a dataclass. But this is a brewaking change in all the SDKs anyway, so maybe might as well make it a breaking proto change?

Reply:
that's fair - I guess there was a feeling that breaking protos is a bit more significant than breaking the client method

Reply:
discussed IRL - we're gonna explicitly deprecate the proto field but leave it in: https://github.com/viamrobotics/api/pull/487